src/java/lambda/others/TableQuery.java

```diff
@@ -0,0 +1,47 @@
+package lambda.others;
+
+import java.util.Iterator;
+
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDB;
+import com.amazonaws.services.dynamodbv2.AmazonDynamoDBClientBuilder;
+import com.amazonaws.services.dynamodbv2.document.DynamoDB;
+import com.amazonaws.services.dynamodbv2.document.Item;
+import com.amazonaws.services.dynamodbv2.document.ItemCollection;
+import com.amazonaws.services.dynamodbv2.document.QueryOutcome;
+import com.amazonaws.services.dynamodbv2.document.Table;
+import com.amazonaws.services.dynamodbv2.document.spec.QuerySpec;
+
+public class TableQuery {
+	
+	private static AmazonDynamoDB dynamoDB;
+	private static DynamoDB dynamo;
+	
+	static {
+		dynamoDB = AmazonDynamoDBClientBuilder.standard()
+		                .withRegion("us-east-1")
+		                .build();
+		dynamo = new DynamoDB(dynamoDB);
+	}
+
+	private String tableName;
+	private QuerySpec spec;
+	
+	public TableQuery(String tableName, QuerySpec spec) {
+		this.tableName = tableName;
+		this.spec = spec;
+	}
+	
+	public String query() {
+		
+    	Table table = dynamo.getTable(tableName);
+    	ItemCollection<QueryOutcome> items = table.query(spec);
+    	
+    	String body = "";
+    	Iterator<Item> iterator = items.iterator();
+    	while (iterator.hasNext()) {
```

Comment:
Nie jestem pewien, ale można by tu zrobić streama

Reply:
Zobaczę co da się z tym zrobić, ale tak jak pisałem refactor lambd zrobię kiedy indziej.
Dużo da się w nich poprawić m.in. zapis wszystkiego do jednej tabeli, hardcodowane json-y czy niepotrzebne pola w klasach ServerlessInput/Output.

Reply:
Ok